src/System/Components/RouterLink.tsx

```diff
@@ -89,12 +89,17 @@ export const RouterLink: React.FC<React.PropsWithChildren<
       }
     }
 
+    const location = router?.createLocation?.((to as string) ?? "") ?? {
+      pathname: to,
+      query: {},
+    }
```

Comment:
Not the prettiest thing but need a fallback for our millions of tests that intersect this component